web/src/pages/NewMeeting.css

```diff
@@ -236,7 +236,7 @@ ul {
 }
 
 button[type="button"]:hover {
-	background-color: #b32d3a;
+	/* background-color: #b32d3a; //this makes the whole background of delete button red*/
```

Comment:
If you don't want this code, delete it!

Reply:
I have checked and ensured that this rule doesn't effect any other elements, and removed the unnecessray commented lines